tests/integration/test_system_containers.sh

```diff
@@ -152,21 +152,32 @@ image_digest=$(ostree --repo=${ATOMIC_OSTREE_REPO} show --print-metadata-key=doc
 ${ATOMIC} images list > images.out
 grep "busybox.*$image_digest" images.out
 
-${ATOMIC} images list > images.out
-${ATOMIC} images list --all > images.all.out
+${ATOMIC} images list -f type=system > images.out
+${ATOMIC} images list -f type=system --all > images.all.out
 test $(wc -l < images.out) -lt $(wc -l < images.all.out)
 grep -q '<none>' images.all.out
-OUTPUT=$(! grep -q '<none>' images.out)
+OUTPUT=$(! grep -c '<none>' images.out)
 if test $OUTPUT \!= 0; then
```

Comment:
I'll revert it, probably the `if test` check later is not needed, as grep would return 0 in case of matches